sys/include/registry/store/registry_store_dummy.h

```diff
@@ -0,0 +1,24 @@
+#ifndef REGISTRY_STORE_DUMMY_H
```

Comment:
I would put these dummy storage only as a test, not under `sys`

Reply:
Done